tests/fluo

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+
+set -e
+
+rm -rf fluo
+git clone https://github.com/fluo-io/fluo.git
+cd fluo
+mvn verify -Daccumulo.version=$1
```

Comment:
fluo will not work w/ Accumulo 1.5.. could add a check to this script to do nothing if version < 1.6